Woof/WoofTests/Model/Preferences/PreferencesHandlerTestHelper.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/**
+  A helper struct for handling preferences in tests.
+ - Note: This struct is intended for use in test cases only.
+ */
+struct PreferencesHandlerTestHelper {
+    /**
+     Cleans the storage used for preferences.
+     This method deletes the user preferences data from the key-value storage.
+
+     Example usage:
+     ```
+     PreferencesHandlerTestHelper.cleanStorage()
+     ```
+     */
+    static func cleanStorage() {
+        let keyValueStorage = KeyValueStorage(KeyValueStorage.Name.preferences)
+        keyValueStorage.deleteData(for: KeyValueStorage.Key.userPreferences)
+    }
+}
```

Comment:
To be honest, it was a confusing file for me when I saw it among the added file :)
You can make it as an extension and call in tests `PreferencesHandler.cleanStorage `

```suggestion
extension PreferencesHandler {
    static func cleanStorage() {
        let keyValueStorage = KeyValueStorage(KeyValueStorage.Name.preferences)
        keyValueStorage.deleteData(for: KeyValueStorage.Key.userPreferences)
    }
}
```

Reply:
I agree, is it worth leaving this extension in the tests folder?

Reply:
As Dersim suggested you can make it as extension for KeyValueStorage